ocdskingfishercolab/__init__.py

```diff
@@ -429,7 +430,7 @@ def get_table_and_path(field, scope_table):
         table = "release_summary"
 
         for num, table_canditate in enumerate(table_candidates):
-            if scope_table == table_canditate:
+            if scope_table[:-8] == table_canditate:
```

Comment:
Please correct all occurrences to `table_candidate`.

Reply:
Done in https://github.com/open-contracting/kingfisher-colab/pull/58/commits/408ec27a28af273db1358326af7f353d81f452d1